NuKeeper.AzureDevOps/AzureDevOpsSettingsReader.cs

```diff
@@ -7,13 +7,42 @@
 
 namespace NuKeeper.AzureDevOps
 {
+    // URL pattern is
+    // https://dev.azure.com/{org}/{project}/_git/{repo}/
     public class AzureDevOpsSettingsReader : ISettingsReader
     {
-        private readonly FileSettings _fileSettings;
+        public Platform Platform => Platform.AzureDevOps;
```

Comment:
This will return a new instance of the enum every time you call the property Platform. This should probably be a constant

Reply:
Tech note: that's not true. A) Enums are (underneath) 32 bit value types,  passed by value and have same 'cost' as passing an e.g. memory address around B) Public constants are dangerous, because they get embedded into compiled user code C) single-op methods are inlined by JIT compiler at runtime anyway, so no need to overo-optimise them.

Reply:
We're generally not worried about this kind of optimisation in NuKeeper. Overhead is minimal and the run-time is dominated by the cost of file system access, remote api requests and running external processes.